raven/src/main/webapp/META-INF/context.xml

```diff
@@ -9,9 +9,9 @@
         factory="org.apache.tomcat.jdbc.pool.DataSourceFactory" closeMethod="close"
         minEvictableIdleTimeMillis="60000" timeBetweenEvictionRunsMillis="30000"
         maxWait="20000"
-        initialSize="0" minIdle="0" maxIdle="${raven.invuser.maxActive}" maxActive="${raven.invuser.maxActive}"
-        username="${raven.invuser.username}" password="${raven.invuser.password}"
-        driverClassName="org.postgresql.Driver" url="${raven.invuser.url}"
+        initialSize="0" minIdle="0" maxIdle="${raven.invadm.maxActive}" maxActive="${raven.invadm.maxActive}"
```

Comment:
This is wrong. raven only queries the database and the intent is that it uses the account with the lowest priv (tapuser)... hence the prop name invuser

Reply:
The raven readme used invadm, and pgdev had an invadm user, but not an invuser. I thought invuser was incorrect. Should've asked. Updated the readme and the context.xml to use invuser.